roles/openshift_logging/files/curator.yml

```diff
@@ -0,0 +1,18 @@
+# Logging example curator config file
```

Comment:
Currently, no there isn't short of editing the actual files provided by the role. 

Are there other roles that we provide that do this already that we can use as inspiration/take advice from?
Would it make sense to provide the path to a file or let users create a configmap that we would read from/reference?